tests/Unit/ValueObject/GalleryValueObjectTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Unit\Patterns;
+
+use Drupal\oe_theme\ValueObject\ImageMediaValueObject;
+use Drupal\oe_theme\ValueObject\GalleryValueObject;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Test file value object.
+ */
+class GalleryValueObjectTest extends UnitTestCase {
```

Comment:
We should change this class name if we change the tested class.

Reply:
:+1: